Tools/WinMLRunner/BindingUtilities.h

```diff
@@ -144,6 +144,35 @@ namespace BindingUtilities
         return elementStrings;
     }
 
+    template <typename T>
+    void PreProcessImageToBinding(const SoftwareBitmap& softwareBitmap, ModelBinding<T>& binding, float scale, const std::array<float, 3>& meanStdDev)
+    {
+        const auto imgHeight = softwareBitmap.PixelHeight();
+        const auto imgWidth = softwareBitmap.PixelWidth();
+        const auto resultSize = imgHeight * imgWidth * 3;
+        if (binding.GetDataBufferSize() != resultSize)
+        {
+            throw hresult_invalid_argument(L"Image Input size/shape is different from what model expects");
+        }
+
+        T* bindingData = binding.GetData();
+        if (scale != 1.0f &&
+            (meanStdDev[0] != 0 || meanStdDev[1] != 0 || meanStdDev[2] != 0))
```

Comment:
Is it impossible that the actual meanStdDev is {0, 0, 0}? why is scale != 1.0f not sufficient?